.readthedocs.yml

```
@@ -1,6 +1,12 @@
 ---
 version: 2
 
+# Patch to get docs building in current pattern
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.11"
```

```suggestion
    python: "3.10"
```

Swapping to 3.10 to align with apps.  If 3.11 please PR into cookie cutter app repo as well. They're not dependent on each other but I recommend for consistency to keep them in line